tasks/rh-sign-image-cosign/rh-sign-image-cosign.yaml

```diff
@@ -24,6 +24,10 @@ spec:
         The relative path in the workspace to a text file that contains a list of repositories
         that needs registry.access.redhat.com image references to be signed (i.e.
         requires_terms=true), one repository string per line, e.g. "rhtas/cosign-rhel9".
+    - name: retries
+      description: Retry cosign N times.
+      type: string
+      default: "0"
```

Comment:
Maybe a default of 3 more practical?

Reply:
Yeah, changed it to 3.

Reply:
Yeah, I agree. That way you won't have to change the pipeline definition.